### PR TITLE
fix(orderedlist): list points in layer2 page

### PR DESCRIPTION
**Pull request by SimardeepSingh1450**
## Description
Made the points fit in center of the circle and fixed the oval shape to circular shape.

Earlier the numbers in points were not in center of the circle and circle was not in proper circle shape, it was in oval shape, fixed both of these problems.

## Related Issue
Bug report: L2 numbered list doesnt have aligned numbers: #9974

### CODE REVIEW

File: src/components/OrderedList.tsx [hunk 1]
```diff
@@ -18,12 +18,12 @@ export interface IProps {
...
-  top: "-2px", // adjusts circle + number up and down
+  top: "-5px", // adjusts circle + number up and down
   left: "-3rem",
-  width: "35px",
-  aspectRatio: "1",
-  height: "2rem",
-  pt: "7px", // adjusts number up and down,
+  width: "32px",
+  // aspectRatio: "1",
+  height: "1.5rem",
```
Comment (nhsz) [inline src/components/OrderedList.tsx:25]:
```suggestion
  height: "1.6rem",
```

File: src/components/OrderedList.tsx [hunk 2]
```diff
@@ -18,12 +18,12 @@ export interface IProps {
...
   content: `counter(li-counter)`,
   position: "absolute",
-  top: "-2px", // adjusts circle + number up and down
+  top: "-5px", // adjusts circle + number up and down
   left: "-3rem",
-  width: "35px",
-  aspectRatio: "1",
-  height: "2rem",
-  pt: "7px", // adjusts number up and down,
+  width: "32px",
```
Comment (nhsz) [inline src/components/OrderedList.tsx:23]:
```suggestion
  width: "34px",
```

File: src/components/OrderedList.tsx [hunk 3]
```diff
@@ -18,12 +18,12 @@ export interface IProps {
 const liCustomType: SystemStyleObject = {
   content: `counter(li-counter)`,
   position: "absolute",
-  top: "-2px", // adjusts circle + number up and down
+  top: "-5px", // adjusts circle + number up and down
```
Comment (nhsz) [inline src/components/OrderedList.tsx:21]:
```suggestion
  top: "-3px", // adjusts circle + number up and down
```

File: src/components/OrderedList.tsx [hunk 4]
```diff
@@ -18,12 +18,12 @@ export interface IProps {
...
   left: "-3rem",
-  width: "35px",
-  aspectRatio: "1",
-  height: "2rem",
-  pt: "7px", // adjusts number up and down,
+  width: "32px",
+  // aspectRatio: "1",
```
Comment (nhsz) [inline src/components/OrderedList.tsx:24]: we should remove this property

**Review by nhsz — CHANGES_REQUESTED** [review]
Hey @SimardeepSingh1450, thanks for the contribution! I left some suggestions to adjust some values to align the circles a bit better

**Comment by SimardeepSingh1450** [comment]
@nhsz thanks for the suggestions, I have commited them in the pr, please review and merge.

**Review by nhsz — APPROVED** [review]
Thanks @SimardeepSingh1450, LGTM!

**Pull request merged**